requirements.txt

```diff
@@ -0,0 +1 @@
+-e .
```

Comment:
Can you explain me what this produce?

Reply:
If you run ```pip install -r requirements.txt```, putting this in the requirements.txt will run ```pip install -e .``` which will get the dependencies from the setup.py file. I think this is nicer than what was there before.